### ISSUE TITLE
(Bug report) '@' Context menu's auto-completion not showing most relevant but most recent

### ISSUE THREAD

**Issue by diyoyo** [TriliumNext/Notes#411]
### TriliumNext Version

0.90.4

### What operating system are you using?

macOS

### What is your setup?

Local + server sync

### Operating System Version

macOS latest + server on Debian 12

### Description

So, I continue with the hardly noticeable changes from zadam's : 
When typing '@' to link a note, it seems like only the most recent notes having the right word are showing up in the context menu list. Sadly, the links that I use most are also the most ancient ones, and now they're no longer proposed...

Also, but this is probably more "enhancement" than bug, I'd love to have an auto-completion/search proposal based solely on the title of the note, not the entire notepath nor any attributes.

**Comment by eliandoran** [comment]
@diyoyo , could you please clarify this point?

>  When typing '@' to link a note, it seems like only the most recent notes having the right word are showing up in the context menu list. Sadly, the links that I use most are also the most ancient ones, and now they're no longer proposed...

I'm on 0.90.5-beta and I could easily find a note from 2014, maybe there is something specific that is causing your issue?
How many results do you get? Maybe there is a limit on the number of results and that's what's causing it.

**Comment by diyoyo** [comment]
@eliandoran yeah that's probably a matter of number of results. The context menu returns 9 results + the "create new" button, while the search field says "233 results".
And yet, there really are only 3 notes that have this exact single-word title.

**Comment by eliandoran** [comment]
@diyoyo , do you think it could be the labels that it's finding? How many characters are you searching for? Honestly we shouldn't even make a search unless you type three characters.
